### PR TITLE
admin/bugfixs-and-py38-support

### PR DESCRIPTION
**Pull request by evamaxfield**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [x] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [x] Provide context of changes.

In trying to resolve `actk`'s failing pipeline due to normalization, @donovanr had the bright idea of actually looking at the normalization code in this package. Which it turns out I was a rookie and changed pretty drastically. Unfortunately since there are no tests (largely) in this repo the change was uncaught.

Additionally when comparing the prior 0.7.2 version and the current 0.7.3 version I found what I believe is a similar change I made in the TextureAtlas generation code. And have reverted that change as well.

Finally, `vtk` now supports py38 so we now support py38 so I am now allowing that to be upgraded in addition to fixing the flake8 lint error.

- [ ] Provide relevant tests for your feature or bug fix.
- [ ] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

File: aicsimageprocessing/textureAtlas.py [hunk 1]
```diff
@@ -207,9 +207,9 @@ def _is_valid_atlas(self, atlas):
         if atlas.atlas.shape is None:
             return False
         shape = atlas.atlas.shape
-        if self.dims.atlas_width != shape[1]:
+        if self.dims.atlas_width != shape[2]:
             return False
-        if self.dims.atlas_height != shape[2]:
+        if self.dims.atlas_height != shape[1]:
```
Comment (donovanr) [inline aicsimageprocessing/textureAtlas.py:212]: height and width getting swapped here?

File: aicsimageprocessing/normalization.py [hunk 2]
```diff
@@ -9,7 +9,7 @@ def normalize_img(img, mask=None, method="img_bg_sub"):
     if mask is None:
         mask = np.ones(img.shape)
 
-    if method != "img_bg_sub":
+    if method == "img_bg_sub":
```
Comment (donovanr) [inline aicsimageprocessing/normalization.py:12]: ✔️

**Review by donovanr — COMMENTED** [review]
`!=` -? `==` looks good, unclear on the dims `1` -> `2` stuff on the `atlas_width` & `atlas_height` stuff.  you sure that's good?

File: aicsimageprocessing/textureAtlas.py (hunk 1 above)
Comment (evamaxfield) [reply aicsimageprocessing/textureAtlas.py:212]: Yea. I tried to link to the actual line in the diff but yea. It definitely got swapped in the last version somehow

**Pull request merged**